web/src/main/webapp/app/app.js

```diff
@@ -421,7 +445,11 @@
     // Get a list of all modules to find all files to load.
     getUser().then(function(data) {
       user = data;
-
+      if (user && user.parameters && user.parameters.launchId !== undefined) {
```

Comment:
Should  '!== undefined' be removed and that you just leave user.parameters.launchId. If it is truthy it will not be undefined, null or empty string.